internal/cli/query/processes.go

```
@@ -0,0 +1,152 @@
+	if len(filter) > 0 {
+		variables = map[string]any{"filter": filter}
+	}
+
+	body, err := json.Marshal(map[string]any{
+		"query":     processesQuery,
+		"variables": variables,
+	})
+	if err != nil {
+		return fmt.Errorf("marshal request: %w", err)
+	}
+	req, err := http.NewRequest(http.MethodPost, "http://"+server.DefaultAddr+"/graphql", bytes.NewReader(body))
```

golangci-lint (2.12.1)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)